src/NServiceBus.AcceptanceTesting/NServiceBus.AcceptanceTesting.csproj

```diff
@@ -17,6 +17,11 @@
     <PackageReference Include="Particular.Packaging" Version="0.9.0" PrivateAssets="All" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
We should instead remove Fody.Janitor and Fody and remove the [SkipWeaving] directive from AcceptanceTestingOutboxTransaction